modules/secret/secret.go

```diff
@@ -19,13 +20,13 @@ import (
 func AesEncrypt(key, text []byte) ([]byte, error) {
 	block, err := aes.NewCipher(key)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("AesEncrypt invalid key: %v", err)
 	}
 	b := base64.StdEncoding.EncodeToString(text)
 	ciphertext := make([]byte, aes.BlockSize+len(b))
 	iv := ciphertext[:aes.BlockSize]
-	if _, err := io.ReadFull(rand.Reader, iv); err != nil {
-		return nil, err
+	if _, err = io.ReadFull(rand.Reader, iv); err != nil {
+		return nil, fmt.Errorf("AesEncrypt unable to read iv: %w", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("AesEncrypt unable to read initialization vector: %w", err)
```

Reply:
IV is already a terminology. So I guess we can call it `IV`

```suggestion
		return nil, fmt.Errorf("AesEncrypt unable to read IV: %w", err)
```